### DISCUSSION TITLE
posts/ygc/

### DISCUSSION THREAD

**Discussion by giscus** [michaelneuper/website#64] [General]
# posts/ygc/

Introduction Yao’s Garbled Circuit is a cryptographic protocol that enables two-party secure computation in which two mistrusting parties can jointly evaluate a function over their private inputs without the presence of a trusted third party.
The invention of garbled circuit was credited to Andrew Yao, as Yao introduced the idea in [FOCS'86]. The first written document about thus technique was by Goldreich, Micali and Wigderson in [STOC'87].
Yao’s protocol solving Yao’s Millionaires’ Problem [IEEE'82] was the beginning example of secure computation, yet it’s not directly relate to garbled circuit.

https://giapppp.github.io/posts/ygc/

**Comment by dvckl3** [comment]
ảnh bị lỗi r a ơi :>
